apps/docs/src/.vitepress/components/ComponentCard.vue

```diff
@@ -0,0 +1,61 @@
+<script lang="ts" setup>
+export type ComponentCardProps = {
+  /** Component name. */
+  name: string;
+  /** If true a "Implemented" badge will be shown, "Coming Soon" otherwise. */
+  implemented?: boolean;
+};
+
+const props = defineProps<ComponentCardProps>();
+</script>
+
+<template>
+  <div class="card">
```

Comment:
the mega deluxe option would be to link the milestones that martin created, at least for the components where we have them already ;)
to give ppl a link to jump to where they will see the progress.

See
https://github.com/SchwarzIT/onyx/milestones

Reply:
Imho thats too much information :D If we want to add it, I'd prefer a separate PR for it

Reply:
oki doki :)